values/loki/loki.gotmpl

```diff
@@ -163,11 +163,18 @@ loki:
         s3: http://otomi-admin:{{ $v.otomi.adminPassword }}@minio.minio.svc.cluster.local.:9000/loki
         s3forcepathstyle: true
       {{- end }}
-      {{- if eq $st.type "S3" }}
+      {{- if eq $st.type "s3" }}
+      {{- if $st.s3.useInstanceRole }}
+      aws:
+        s3: s3://{{ $st.s3.region }}/{{ $st.s3.bucket }}
+        dynamodb:
```

Comment:
Why is `dynamodb:` is needed here?

Reply:
https://grafana.com/docs/loki/latest/configure/examples/#3-s3-without-credentials-snippetyaml